ui-core/src/participant/landing/sections/FrequentlyAskedQuestionsTemplate.tsx

```diff
@@ -165,7 +173,7 @@ function FrequentlyAskedQuestionsTemplate(props: FrequentlyAskedQuestionsProps)
             <span className="d-inline-block me-2 text-center" style={{ width: 20 }}>
               <FontAwesomeIcon icon={allQuestionsAreExpanded ? faXmark : faPlus} />
             </span>
-            {allQuestionsAreExpanded ? 'Collapse' : 'Expand'} all
+            {allQuestionsAreExpanded ? collapseAllText : expandAllText}
```

Comment:
These were the only 2 bits of text that users couldn't already override in the siteContent JSON

Reply:
It still said "Expand All" on the FAQ page for all languages for me - is that expected?

Reply:
These are overridable in the site content editor rather than being a part of LanguageTexts, so they'll still say Expand All by default